FAQ.md

```diff
@@ -24,11 +25,19 @@ This demo site contains mostly all known plugins ; i.e. the official ones + the
 
 root / root
 
-## 3. Docker image?
+## 3. What's limitation of password?
+
+GitBucket has a limitation of password as below.
+
+- Use alphanumeric character
```

Comment:
`-`, `_` and `.` are also allowed in password.

Reply:
Modified again.
Could you confirm it ?